.golangci.yaml

```diff
@@ -29,16 +29,13 @@ output:
 linters:
   disable-all: true
   enable:
-    - deadcode
```

Comment:
Can I ask why these should get disabled? (just curiosity)

Reply:
They've been deprecated :) 
<img width="1162" alt="image" src="https://user-images.githubusercontent.com/17353640/211003470-c9ce93de-9dd1-4b7d-be51-81078b852328.png">